ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -30,9 +43,33 @@
     group: root
     mode: 0644
```

Comment:
I think that this may be a case of bad task names. Both systems get the SAME file (crane.conf)  in /etc/crane.conf
```
[general]
data_dir: /var/lib/pulp/published/docker
debug: false
```
Which is put in place by the `Install Crane configuration file` task

Only el6 gets `crane_el6.wsgi` in `/usr/share/crane/crane_el6.wsgi` which has the task name `Install Crane configuration file (RHEL 6)`, but it is totally different/has different purpose.  I'll change it to `Install Crane WSGI config file (RHEL6)`.
```
sys.path.insert(0, '/usr/lib/python2.6/site-packages/Jinja2-2.6-py2.6.egg')

from crane.wsgi import application
```